util/job_manager.rb

```diff
@@ -83,6 +84,7 @@ def perform
         p[:result].first[:elapsed] = p[:graph].last[:time]
       end
       Result.set(self.uuid, p)
+      Models.delete_all
```

Comment:
Do we need this delete? Worker dies after completing the process, right?

Reply:
Thanks for pointing it out, it is a residual line. I will remove it

Reply:
This delete came back to life as well :zombie: